packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -136,14 +143,21 @@ export class AddJob {
     );
 
     if (delay) {
-      Logger.verbose(
-        'Delay is active, Creating execution details',
-        LOG_CONTEXT
-      );
+      const logMessage =
+        job.type === StepTypeEnum.DELAY
+          ? 'Delay is active, Creating execution details'
+          : job.type === StepTypeEnum.DIGEST
+          ? 'Digest is active, Creating execution details'
+          : 'Unexpected job type, Creating execution details';
```

Comment:
made it like this and not with template string so it will be easier to discover this log.

Reply:
the unexpected backoff should not happen added it just in case for the future. we can remove it if it looks complex.